examples/gcp-pubsub-quickstart/pom.xml

```diff
@@ -47,6 +47,11 @@
       <artifactId>smallrye-reactive-messaging-gcp-pubsub</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
```

Comment:
What's the dependency for?

Reply:
Actually I was going to ping you about this one. Seems like GCP is now using (yet another) tracing library.

Reply:
Weird